macbuild.tool

```diff
@@ -68,7 +69,7 @@ package() {
   popd || exit 1
 }
 
-cd $(dirname "$0")
+cd "$(dirname $0)" || exit 1
```

Comment:
This may not work if `$0` contains spaces as far as I know.

Reply:
Indeed.

On review, issue is actually that command expansion is not needed as not assigning output.

Will modify to `cd dirname "$0" || exit 1`